test/math_test.py

```diff
@@ -1577,6 +1577,13 @@ def test_pickle(self):
         self.assertEqual(pickle.loads(pickle.dumps(v2)), v2)
         self.assertEqual(pickle.loads(pickle.dumps(v3)), v3)
 
+    def testVector2Subclass(self):
+        class Vector(pygame.math.Vector2):
+            pass
+        v = Vector(2.0,2.0)
```

Comment:
It's usual to put a space after arguments. For example: `v = Vector(2.0, 2.0)`

Reply:
Okay thanks, I'll be sure to do that next time.

Reply:
The pygame code isn't the cleanest yet, or using modern conventions everywhere. But eventually we will move to using [pylint](https://www.pylint.org/) to help catch these types of issues.